src/webots/nodes/utils/WbVisualBoundingSphere.cpp

```diff
@@ -40,31 +40,6 @@ void WbVisualBoundingSphere::enable(bool enabled, const WbBaseNode *node) {
 void WbVisualBoundingSphere::createSphere(
```

Comment:
Shouldn't we totally get rid of this dumb method?
The same question applies also to `WbVisualBoundingSphere::show()` and `WbVisualBoundingSphere::clear()`.

Reply:
these methods were very useful for debugging.
I don't know if now with wren there is a better method to debug the bounding spheres.
My idea was to restore these methods, but not in this PR.

Reply:
OK, in that case the comment shouldn't be TODO_WREN, but some explanation saying that these methods are useful and should be uncommented for debugging purposes.

Reply:
The problem is that they have not been ported to WREN, this is why there is the `TODO_WREN`.